suse_openstack_cloud

```diff
@@ -250,6 +250,16 @@ find_and_pconf_files /srv/tftpboot/discovery/pxelinux.cfg -type f
 find_and_pconf_files /srv/tftpboot/nodes                  -type f
 
 #############################################################
+section_header "Crowbar database"
+
+crowbar_db_dump_dir=/var/log/crowbar/db-dump
+# dump the database tables
+sqlite3 /opt/dell/crowbar_framework/db/production.sqlite3 .dump > $crowbar_db_dump_dir/production.dump.sql
```

Comment:
@toabctl good thought, but I am not aware of a rails built in way to do that. I guess one would have to write a custom rake task for it (with querying `ActiveRecord::Base`). there is a `rake db:schema:dump` task, but it wouldn't contain the data, just the schema.